frontend/packages/console-shared/src/components/getting-started/GettingStartedGrid.tsx

```diff
@@ -55,10 +56,10 @@ export const GettingStartedGrid: React.FC<GettingStartedGridProps> = ({ onHide,
   );
 
   return (
-    <Card className="ocs-getting-started-grid">
+    <Card className="ocs-getting-started-grid" data-test="card getting-started">
```

Comment:
This ID could clash with the dynamic IDs of the add actions cards.

Reply:
I think that's a feature, not a bug. If someone creates a dynamic "Getting Started" card, it's okay that our **tests** fail.

"For the tests" (visual) this is a card.

Reply:
Removed the card for now. So tests need to check for `getting-started`. /cc @gajanan-more